### PR TITLE
Improve MMM Docs

### PR DESCRIPTION
**Pull request by juanitorduz**
Improve MMM docs, in particular docstrings in the key classes.

----
📚 Documentation preview 📚: https://pymc-marketing--612.org.readthedocs.build/en/612/

### CODE REVIEW

File: pymc_marketing/mmm/delayed_saturated_mmm.py [hunk 1]
```diff
@@ -706,13 +715,129 @@ class DelayedSaturatedMMM(
...
+            control_columns=[
+                "event_1",
+                "event_2",
+                "t",
+            ],
+            adstock_max_lag=8,
+            yearly_seasonality=2,
+        )
+
+    Observe can choose the distribution and all parameters of the model.
```
Comment (twiecki) [inline pymc_marketing/mmm/delayed_saturated_mmm.py:813]: Observe can choose?

Comment (juanitorduz) [reply pymc_marketing/mmm/delayed_saturated_mmm.py]: Ups! Thanks! I re-phrased it :)

File: pymc_marketing/mmm/delayed_saturated_mmm.py [hunk 2]
```diff
@@ -810,7 +810,7 @@ class DelayedSaturatedMMM(
             yearly_seasonality=2,
         )
 
-    Observe we can choose the distribution and all parameters of the model.
+    We can choose the distribution and all parameters and likelihood of the model through the `model_config` parameter.
```
Comment (twiecki) [inline pymc_marketing/mmm/delayed_saturated_mmm.py:813]: "As you can see, we can configure all prior and likelihood distributions via the `model_config`."?

Comment (juanitorduz) [reply pymc_marketing/mmm/delayed_saturated_mmm.py]: 👍 Added!

**Review by twiecki — APPROVED** [review]

**Pull request merged**